crates/frontend/src/components/context_card.rs

```diff
@@ -31,9 +31,10 @@ pub fn context_card(
         .clone()
         .into_iter()
         .map(|(k, v)| {
-            let k = Value::String(k.trim_matches('"').to_string());
-            let v = Value::String(format!("{}", v).trim_matches('"').to_string());
-            Map::from_iter(vec![(String::from("KEY"), k), (String::from("VALUE"), v)])
+            Map::from_iter(vec![
+                (String::from("KEY"), Value::String(k.clone())),
+                (String::from("VALUE"), v),
+            ])
```

Comment:
was this tested? Or can we remove this because of @sauraww 's change earlier?

Reply:
This is tested, I am just getting rid of trim logic, as `html_display` will take care of showing these values without quotes in frontend.